File: solutions/typescript/darts/1/darts.ts
export function score(x: number, y: number): number {
  let score = 0;
  const r = Math.sqrt(x * x + y * y );
  
  if (r > 10) {
    score = 0;
  } else if (r > 5 && r <= 10) {
    score = 1;
  } else if (r > 1 && r <= 5) {
    score = 5;
  } else if (r <= 1) {
    score = 10;
  }

  return score;
}

File: solutions/python/darts/1/darts.py
def score(x, y):
    radius = (x ** 2 + y ** 2) ** 0.5
    if radius <= 1 :
        result_score = 10
    elif radius <= 5:
        result_score = 5
    elif radius <= 10:
        result_score = 1
    elif radius > 10:
        result_score = 0

    return result_score
